httpmw/internal/ddlog/log.go

```diff
@@ -15,7 +16,8 @@ type HTTPRequest struct {
 }
 
 type URLDetails struct {
-	Path string `json:"path"`
+	Path        string         `json:"path"`
+	QueryString map[string]any `json:"query_string,omitempty"`
```

Comment:
Should be `queryString`, not `query_string`, according to [Datadog docs](https://docs.datadoghq.com/standard-attributes/?search=http.url_details). Sadly no specific info about the contents; traefik logs a `map[string]string` with only the last value if multiple values with the same key are present (though the full query string is in `.RequestPath`)